cmake/firebase_version.sed

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# USAGE:
+# sed -n -f cmake/firebase_version.sed FirebaseCore.podspec
+
+# Find the line assigning to s.version
+/.*Firebase_VERSION=/ {
+  # Remove the assignment
```

Comment:
Why?

Reply:
I've added more comments to make this clearer.